app/javascript/packages/components/index.ts

```diff
@@ -12,6 +12,8 @@ export { default as IconListItem } from './icon-list/icon-list-item';
 export { default as IconListTitle } from './icon-list/icon-list-title';
 export { default as Link } from './link';
 export { default as PageFooter } from './page-footer';
+export { default as LocationCollection } from './location-collection';
+export { default as LocationCollectionItem } from './location-collection-item';
```

Comment:
(Non-blocking) The `components` package is intended to include only general-purpose, reusable components, but this feels feature-specific to in-person proofing. I might suggest either moving it into `document-capture` or possibly even creating a new package for IPP-specific code.

Reply:
Follow-up at https://github.com/18F/identity-idp/pull/6756